Userland/Games/BrickGame/BrickGame.cpp

```diff
@@ -281,16 +282,39 @@ class Bricks final {
         return m_well[pos] || m_block[pos];
     }
 
-    [[nodiscard]] RenderRequest rotate_left() { return set_current_block(Block(m_block).rotate_left()); }
+    [[nodiscard]] RenderRequest rotate_left() {
+        if (paused()) {
+            return RenderRequest::SkipRender;
+        }
```

Comment:
Instead of checking for `paused()` in each method, why not prevent calling this method at all? You can check for `paused()` just once in `keydown_event` and prevent these methods from being called in the first place.

Reply:
I check for paused in each method instead of in the keydown_event to reduce dependency between the two classes. That way in the future there could be different game modes that manage pausing differently.

Reply:
Sure - in that case, please remove the braces for all these one-line if-blocks.